packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -205,18 +221,18 @@ export const handleFiatInputChange = (outputId: number, fiatValue: string) => (
     dispatch: Dispatch,
     getState: GetState,
 ) => {
-    const { account } = getState().wallet.selectedAccount;
+    const { account, network } = getState().wallet.selectedAccount;
     const { fiat, send } = getState().wallet;
-
-    if (!account || !fiat || !send) return null;
+    if (!account || !fiat || !send || !network) return null;
 
     const output = getOutput(send.outputs, outputId);
     const fiatNetwork = fiat.find(item => item.symbol === account.symbol);
-    if (!fiatNetwork) return null;
+
+    if (!fiatNetwork || !network) return null;
```

Comment:
!network condition is not necessary here, since it was already checked in line 226

Reply:
selectedAccountReducer 

![image](https://user-images.githubusercontent.com/3112191/66914737-c1550780-f017-11e9-8a4b-a29c97baea37.png)